### PR TITLE
Added EC-CUBE

### PR DESCRIPTION
**Pull request by Yangsin**
EC-CUBE is the most popular shopping cart system in Japan.  
EC-CUBE is using Silex since version 3.0. 
refs https://github.com/EC-CUBE/EC-CUBE

### CODE REVIEW

File: projects/en/ec-cube.html.twig [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{% block description %}
+    <p>
+
```
Comment (OskarStark) [inline projects/en/ec-cube.html.twig]: please remove the blank line

File: projects/_projects.yml [hunk 2]
```diff
@@ -72,3 +72,4 @@ minor:
     - morfeu
     - pimcore
     - csbill
+    - ec-cube
```
Comment (OskarStark) [inline projects/_projects.yml]: should be `eccube` instead `ec-cube`

File: projects/ec-cube.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+name: ec-cube
```
Comment (OskarStark) [inline projects/ec-cube.yml]: same here

**Comment by Yangsin** [comment]
@OskarStark  Thanks your review, fixed it.

**Comment by OskarStark** [comment]
Thank you @Yangsin 

Status: Reviewed

**Pull request merged**